keylime/tenant.py

```diff
@@ -156,6 +160,19 @@ def __init__(self) -> None:
     def verifier_base_url(self) -> str:
         return f"{bracketize_ipv6(self.verifier_ip)}:{self.verifier_port}"
 
+    def set_full_id_log_msg(self) -> None:
```

Comment:
This is a setup method that needs to be called before logging, correct? The name of the method makes it seems like it's actually logging a message.